doc/onboarding.md

```diff
@@ -169,6 +169,7 @@ Landing a PR
     * `Reviewed-By: human <email>`
       * Easiest to use `git log` then do a search
       * (`/Name` + `enter` (+ `n` as much as you need to) in vim)
+      * Only include collaborators who have commented "LGTM"
```

Comment:
How about expanding it as well?
* Only include collaborators who have commented "LGTM" (Looks Good To Me)